Sources/Controllers/2.0/OAGPXListViewController.mm

```diff
@@ -894,249 +811,44 @@ - (void)onImportClicked
-            [cell.closeButton addTarget:self action:@selector(cancelRoutePressed) forControlEvents:UIControlEventTouchUpInside];
-        }
-        
-        if (cell) {
-            
-            //[cell.iconView setImage: [UIImage imageNamed:@"icon_gpx_fill"]];
-            
-            [cell.titleView setText:[_routeItem getNiceTitle]];
-            
-            double distance = [OAGPXRouter sharedInstance].routeDoc.totalDistance;
-            NSTimeInterval duration = [[OAGPXRouter sharedInstance] getRouteDuration];
-            
-            [cell setDistance:distance wptCount:_routeItem.wptPoints tripDuration:duration];
-        }
-        return cell;
-    }
-    else if (indexPath.section == _tripsSectionIndex)
-    {
-        static NSString* const reusableIdentifierPoint = @"OAGPXTableViewCell";
-        
-        OAGPXTableViewCell* cell;
-        cell = (OAGPXTableViewCell *)[self.gpxTableView dequeueReusableCellWithIdentifier:reusableIdentifierPoint];
-        if (cell == nil)
-        {
-            NSArray *nib = [[NSBundle mainBundle] loadNibNamed:@"OAGPXCell" owner:self options:nil];
-            cell = (OAGPXTableViewCell *)[nib objectAtIndex:0];
-        }
-        
-        if (cell)
-        {
-            OAGPX* item = [self.gpxList objectAtIndex:indexPath.row];
-            [cell.textView setText:[item getNiceTitle]];
-            [cell.descriptionDistanceView setText:[_app getFormattedDistance:item.totalDistance]];
-            [cell.descriptionPointsView setText:[NSString stringWithFormat:@"%d %@", item.wptPoints, [OALocalizedString(@"gpx_waypoints") lowercaseStringWithLocale:[NSLocale currentLocale]]]];
-            
-            cell.accessoryView = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"menu_cell_pointer.png"]];
-            
-            if (_viewMode == kAllTripsMode && [_visible containsObject:item.gpxFileName])
-                [cell.iconView setImage:[UIImage imageNamed:@"menu_cell_selected.png"]];
-            else
-                [cell.iconView setImage:nil];
-        }
-        return cell;
-    }
-    else if (indexPath.section == _createTripSectionIndex)
-    {
-        static NSString* const reusableIdentifierPoint = @"OAIconTextTableViewCell";
-        
-        OAIconTextTableViewCell* cell;
-        cell = (OAIconTextTableViewCell *)[self.gpxTableView dequeueReusableCellWithIdentifier:reusableIdentifierPoint];
-        if (cell == nil)
-        {
-            NSArray *nib = [[NSBundle mainBundle] loadNibNamed:@"OAIconTextCell" owner:self options:nil];
-            cell = (OAIconTextTableViewCell *)[nib objectAtIndex:0];
-        }
-        
-        if (cell) {
-            [cell.textView setText:OALocalizedString(@"create_new_trip")];
-            [cell.iconView setImage: [UIImage imageNamed:@"icon_info"]];
-        }
-        return cell;
-    }
-    else
+    NSString *fileName = [NSString stringWithFormat:@"Trip_%@.gpx", [fmt stringFromDate:[NSDate date]]];
```

Comment:
Since you're changing Trip to track it would be a great idea to replace all occurrences of "trip" throughout the entire project to avoid confusion in the UI.